nextflow.config

```diff
@@ -133,15 +132,14 @@ dag {
     file    = "${params.tracedir}/pipeline_dag_${trace_timestamp}.html"
 }
 
-// TODO: Update the description
 manifest {
     name            = 'Arcadia-Science/hifi2genome'
     author          = 'Arcadia Science'
     homePage        = 'https://github.com/Arcadia-Science/hifi2genome'
     description     = 'Assemble PacBio Hifi reads into a genome and perform QC stats'
     mainScript      = 'main.nf'
     nextflowVersion = '!>=21.10.3'
-    version         = '1.0dev'
+    version = '2.7.2'
```

Comment:
why this v particular version? 

also spacing! (i actually don't really care about the way nf-core does this, but better consistent than not)

Reply:
huh this is interesting, I didn't add this. I think this is actually the version of nf-core tools because I updated last week to this exact version, and I didn't noticed it changed this.